.github/workflows/ci-pipeline.yml

```diff
@@ -42,14 +40,5 @@ jobs:
           target: production
           tags: framasw/todo-app:latest,framasw/todo-app:${{ github.sha }}
 
-      - name: Login to Heroku
-        run: heroku container:login
-
-      - name: Tag Docker image for Heroku
-        run: docker tag framasw/todo-app:latest registry.heroku.com/framasw-todo-app/web
-
-      - name: Push to Heroku
-        run: docker push registry.heroku.com/framasw-todo-app/web
-
-      - name: Release on Heroku
-        run: heroku container:release -a framasw-todo-app web
\ No newline at end of file
+      - name: Call Azure webhook
+        run: curl -dH -X POST ${{ secrets.AZURE_WEBHOOK_URL }}
```

Comment:
One nice addition here can be adding the [--fail](https://curl.se/docs/manpage.html#-f) flag, as otherwise curl reports success as long as a successful connection was made, even if the response reports a failure (e.g. a status code > 400) which can lead to confusing pipeline results

Bonus comment, I can't see a pipeline run in which this has executed so I'm willing to trust it will work but escaping the variables can cause confusion so worth checking carefully that this prints the right output (if it just says it runs for 1 second and prints nothing Azure-related then it probably failed even if it says otherwise!) - if you do hit issues my go to suggestion is:
* Make sure your secret is exactly what you want, with no escaping
* Convert it to an environment variable in an `env` block, which will look something like below (note the important quotes):
```yml
env:
    AZURE_WEBHOOK_URL: "${{ secrets.AZURE_WEBHOOK_URL }}"
```
* Use it as an environment variable rather than a secret:
```
run: curl -dH -X POST $AZURE_WEBHOOK_URL
```

You may well find that bit above unnecessary, but some people run into issues with escaping that can be a pain to debug so just an option if that does hit you